lib/honeybadger/plugins/rails.rb

```diff
@@ -71,16 +71,18 @@ def self.source_ignored?(source)
       end
 
       Plugin.register :rails do
-        requirement { config.load_plugin_insights?(:rails_metrics) && defined?(::Rails.application) && ::Rails.application }
```

Comment:
This should have been `:rails` and not `:rails_metrics`, but this is more of a naming thing and wouldn't have broken anything.